package.json

```diff
@@ -6,6 +6,16 @@
   "bin": {
     "percy": "./bin/run"
   },
+  "watch": {
+    "build": {
+      "patterns": [
+        "src"
```

Comment:
I think this should be limited to just `/src/percy-agent-client`

The rest of the source code does not need to build because `oclif` uses `ts-node` which is able to execute TypeScript directly.

More info over at https://oclif.io/docs/faqs#should-i-use-typescript-or-javascript

Reply:
Ah, tricky. Fixed.